cookbooks/.rubocop_todo.yml

```diff
@@ -1,11 +1,19 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2017-07-06 12:14:14 -0400 using RuboCop version 0.49.1.
+# on 2017-08-17 17:34:59 -0400 using RuboCop version 0.49.1.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
+# Offense count: 1
```

Comment:
Ok, regarding rubocop, I think this offence shouldn't actually have been added, since it was fixed at the same time. I suggest backing out the changes to this file, since leaving it there will mean that we still get one future offence for free in mercurial.rb.